modules/Microsoft.Web/staticSites/config/readme.md

```diff
@@ -0,0 +1,42 @@
+# Web StaticSites Config `[Microsoft.Web/staticSites/config]`
+
+This module deploys Web StaticSites Config.
+// TODO: Replace Resource and fill in description

```

Comment:
I'd suggest the following unless additional description is useful to be added.
```suggestion
This module deploys Web Static Sites Config.

```

Reply:
I would suggest to either use the API name (`StaticSites`) or the product name (`Static Web App`).

Reply:
Why was this comment resolved? I can still only see the placeholder.

Reply:
My suggestion:
```suggestion
This module deploys a Static Site Config.
```